assets/home/components/TopNewsOneByOneCard.jsx

```diff
@@ -41,14 +41,15 @@ const getTopNewsPanel = (item, picture) => {
     </div>);
 };
 
-function TopNewsOneByOneCard({items, title}) {
+function TopNewsOneByOneCard({items, title, product}) {
     return (
         <div className='row'>
             <div className='col-6 col-sm-8'>
                 <h3 className='home-section-heading'>{title}</h3>
             </div>
             <div className='col-6 col-sm-4 d-flex align-items-start justify-content-end'>
-                <button type='button' className='btn btn-outline-primary btn-sm mb-3'>{gettext('More news')}</button>
+                {product &&
+                <button onClick={() => window.location.href = `/wire?q=${getProductQuery(product)}`} type='button' className='btn btn-outline-primary btn-sm mb-3'>{gettext('More news')}</button>}
```

Comment:
same here, actually looks like this could be a component it's duplicated in a few cards already